stock_picking_auto_create_lot/tests/test_stock_picking_auto_create_lot.py

```diff
@@ -178,7 +168,15 @@ def test_multiple_sml_for_one_stock_move(self):
         self.picking.action_assign()
         self.picking.move_line_ids.qty_done = 25.0
         # new sml with 25.0 units
-        self.picking.move_line_ids.copy()
+        self.picking.move_line_ids.copy({"qty_done": 25.0})
         self.picking.button_validate()
         lots = self.picking.move_line_ids.lot_id
         self.assertEqual(len(lots), 2)
+
+    def _assign_manual_serials(self, moves):
+        # Assign manual serials
+        moves.picking_id._set_auto_lot()
+        moves.move_line_ids.qty_done = 1.0
+        moves.picking_id.move_lines._set_quantities_to_reservation()
```

Comment:
IMO, this line should not be inside this method as it blurs the scope and could be confusing. The equivalent step should be inserted into the relevant parts of the test respectively (as suggested).
```suggestion
```